[project_name]/blog.md

```diff
@@ -1,46 +0,0 @@
-# [Replace with Project Title]
```

Comment:
You've accidentally deleted the template file [project_name]/blog.md. You'll need to bring that back before this can be merged. It is now titled [ProjectName]/readme.md.